### ISSUE TITLE
→ Accessibility: Components | Vertical mosaic

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#2255]
## Description

Add a11y definition to the Vertical mosaic component:

* Default role of Vertical mosaic component should be unordered list
* Role can be configured
* Reading order in current implementation differs from proposed

https://jira.tid.es/browse/DSNCORE-1360 

## Tasks

- [x] [Update specs](https://www.figma.com/design/4GmsXwcMOapuCa0Atm6E84/branch/D8fG9SCwrKZNzanOvDqnQP/%F0%9F%94%B8-Mosaic-Specs?m=auto&node-id=451-721&t=RWlPii3Z9u2GkdE4-1)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2287)
